### PR TITLE
Added types for atheris

### PR DESCRIPTION
**Pull request by radarhere**
Adding types for https://github.com/google/atheris

### CODE REVIEW

**Comment by radarhere** [comment]
Let me know if there is any further work that would help this progress.

**Review by srittau — COMMENTED** [review]
Thanks! A quick note: We now require a short comment, explaining why `Any` is necessary. Could you add them?

**Comment by radarhere** [comment]
Sure. I've added a comment for one, and figured out alternatives for the others.

File: stubs/atheris/atheris/utils.pyi [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from typing import IO
+
+def path() -> str: ...
+
+class ProgressRenderer:
+    def __init__(self, stream: IO[str], total_count: int) -> None: ...
```
Comment (srittau) [inline stubs/atheris/atheris/utils.pyi:6]: `IO` and other concrete classes should be avoided in argument annotations. This protocol should work instead:

```python
@type_check_only
class _Writer(Protocol):
    def isatty(self) -> bool: ...
    def write(self, content: bytes, /) -> object: ...
    def flush(self) -> object: ...
```

File: stubs/atheris/atheris/version_dependent.pyi [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+CONDITIONAL_JUMPS: list[str]
+UNCONDITIONAL_JUMPS: list[str]
+ENDS_FUNCTION: list[str]
+HAVE_REL_REFERENCE: list[str]
+HAVE_ABS_REFERENCE: list[str]
+REL_REFERENCE_IS_INVERTED: list[str]
+
+def rel_reference_scale(opname: str) -> int: ...
+
+REVERSE_CMP_OP: list[int]
```
Comment (srittau) [inline stubs/atheris/atheris/version_dependent.pyi:13]: These constants should all probably marked `Final`.

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks! I didn't compare to the implementation, but apart from the things below, everything looks reasonable.

**Review by srittau — APPROVED** [review]

**Comment by srittau** [comment]
I'm sorry for the long delay before we managed to review and merge it.

**Comment by radarhere** [comment]
No worries, I understand there are many other PRs that come in.

Thanks very much

**Pull request merged**